packages/nextjs/app/page.tsx

```diff
@@ -20,12 +35,28 @@ const Home: NextPage = () => {
             <span className="block text-4xl font-bold">Batch 2</span>
           </h1>
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
-          <p className="text-lg flex gap-2 justify-center">
+          <p className="text-lg flex gap-2 mt-8 justify-center">
             <span className="font-bold">
               Checked in builders count:{" "}
               {checkInCount?.toString() ?? <span className="loading loading-spinner loading-sm"></span>}
             </span>
           </p>
+          <p className="text-lg flex gap-2 justify-center">
+            {allowed == undefined || contract == undefined ? (
```

Comment:
NITPICK: 
Any reason to use double `=` instead of using triple `=` for comparison ? 

Its always [good practice](https://arc.net/l/quote/zoduixts) to use strict equal, rather than double equal untill we have very good reason to use it 🙌